Doc/library/sys.rst

```diff
@@ -440,6 +441,9 @@ always available.
       Added ``dev_mode`` attribute for the new :option:`-X` ``dev`` flag
       and ``utf8_mode`` attribute for the new  :option:`-X` ``utf8`` flag.
```

Comment:
If you want to change these two (and line 424-425) for consistent style :)

Reply:
I chose to not change these on purpose, it would be an unrelated change. Someone can work on a PR to update these once this PR is merged ;-)